### PR TITLE
remove amq from webapp

### PR DESCRIPTION
**Pull request by pb82**
AMQ is now AMQ Online but the Webapp was still trying to provision AMQ. This PR removes AMQ and updates the mock config. The Webapp should work in dev mode `yarn start:dev` as well as on a cluster.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -287,7 +287,7 @@ function getMockConfigData() {
         },
         {
           spec: {
-            clusterServiceClassExternalName: 'amq-broker-72-persistence'
+            clusterServiceClassExternalName: 'amq-online-standard'
```
Comment (sedroche) [inline server.js]: I was expecting all removals. Why do we point at a different AMQ?

**Comment by sedroche** [comment]
Verified with https://github.com/integr8ly/installation/pull/349

File: server.js (hunk 1 above)
Comment (pb82) [reply server.js]: AMQ has been replaced with AMQ Online (which as far as i understand comes with the cluster by default).

Comment (sedroche) [reply server.js]: :+1:

Comment (aidenkeating) [inline server.js]: Looks like there's two `amq-online-standard`s listed here now?

Comment (pb82) [reply server.js]: right, i'll remove that. It's the mock config, so can be verified with `yarn start:dev`

**Pull request merged**